### DISCUSSION TITLE
A display exception when  opening directory by NERDTree feature

### DISCUSSION THREAD

**Discussion by Aimerny** [JetBrains/ideavim#582] [Q&A]
When I press the `i` key against the directory with NERDTree plug, idea will split a new window, but I can't do anything with it. unless I close all tabs.
![Snipaste_2023-01-04_21-18-20](https://user-images.githubusercontent.com/121629148/210563611-25d231ed-7f51-4963-a2a4-5704793abaeb.png)

**Comment by AlexPl292** [comment]
Thank you for the report! Please follow this ticket for updates [VIM-2834](https://youtrack.jetbrains.com/issue/VIM-2834)

**Reply by Aimerny** [reply]
Thank you, but I found that the version of my idea may be relatively low, so the highest version of ideavim is only 1.2. When I upgrade ideaVim to 2.0.0, this problem will no longer exist.
